Dockerfile

```diff
@@ -23,7 +23,4 @@ RUN chown -R app:app ./
 
 USER app
 
-VOLUME /code
-WORKDIR /code
```

Comment:
True. Someone mentioned that to me before, I forgot and thought I had added those.
What about moving everything from `/usr/src/app` to `/code`, though?